### ISSUE TITLE
Manifest build fails with error `Field does not exist: encodeUTF8`

### ISSUE THREAD

**Issue by juulSme** [prometheus-operator/kube-prometheus#738]
building the example.jsonnet fails:
```bash
+ xargs '-I{}' sh -c 'cat {} | gojsontoyaml > {}.yaml' -- '{}'
RUNTIME ERROR: Field does not exist: encodeUTF8
        vendor/grafana/grafana.libsonnet:108:74-88      thunk from <object <anonymous>>
        <std>:1022:19-24        thunk from <thunk <bytes> from <function <anonymous>>>
        <std>:1022:10-25        thunk <bytes> from <function <anonymous>>
        <std>:1060:61-66        thunk from <thunk <sanity> from <function <anonymous>>>
        <std>:765:15-18 thunk from <function <anonymous>>
        <std>:761:28-31 thunk from <function <aux>>
        <std>:761:17-32 function <aux>
        <std>:765:5-28  function <anonymous>
        <std>:1060:20-73        thunk <sanity> from <function <anonymous>>
        <std>:1061:9-15 function <anonymous>
        vendor/grafana/grafana.libsonnet:(108:63)-(111:19)      object <anonymous>
        During manifestation
```

**How to reproduce it (as minimally and precisely as possible)**:
`jb install github.com/coreos/kube-prometheus/jsonnet/kube-prometheus@release-0.4`
`./build.sh my_build.jsonnet`

Removing the line `+ { ['grafana-' + name]: kp.grafana[name] for name in std.objectFields(kp.grafana) }` at the end fixes the issue but obviously excludes grafana from the result.

**Comment by huangtianhua** [comment]
I aslo met this issue,  I exported an json file of my dashboard from grafana UI, then I edit the jsonnet followed the docs, when I executed `./build.sh my-test.jsonnet` then this error raised, not sure how to fix this? Could anyone can help? Thanks very much.

**Comment by huangtianhua** [comment]
According jsonnet docs:
std.encodeUTF8(str)
Available since version 0.13.0.

Maybe it's the reason ?

**Comment by huangtianhua** [comment]
I remove the old jsonnet `snap remove jsonnet` and install a new one by `go get github.com/google/go-jsonnet/cmd/jsonnet`  the version is v0.16.0 and build is ok.
